examples/03_matrix_vector_product_mixedprec.cpp

```diff
@@ -4,8 +4,9 @@
 // This must be defined before including any mdspan headers.
 #define MDSPAN_USE_PAREN_OPERATOR 1
 
+#include <experimental/mdspan>
+#include "experimental/__p2630_bits/submdspan.hpp"
```

Comment:
@dalg24 It didn't build (with GCC 11.4 + CMake 3.22.1, otherwise default settings) otherwise.  I really just did this:

```
cmake ../../../src/kokkos/stdBLAS -DLINALG_ENABLE_TESTS=ON -DLINALG_ENABLE_EXAMPLES=ON
make -j12
```

and got errors about not finding `submdspan`.  Note that there's no `<experimental/submdspan>` header to include.

Reply:
I think the issue might be that we will were meant to include `<mdspan/mdspan.hpp>`

https://github.com/kokkos/mdspan/blob/fddbcb85fcff3b450593669ef40914f09cf40f71/include/mdspan/mdspan.hpp#L37-L40

Reply:
None of the following headers exist.

* `experimental/submdspan`
* `experimental/submdspan.hpp`
* `mdspan/submdspan`
* `mdspan/submdspan.hpp`

Reply:
Shouldn't `<mdspan/mdspan.hpp>` include the submdspan headers?